### DISCUSSION TITLE
Missing Beatmaps after copy/pasting old lazer folder.

### DISCUSSION THREAD

**Discussion by X-Vaskys** [ppy/osu#17756] [Q&A] [answered]
I'm unsure if this issue has been brought up already as I couldn't seem to find anything through the search bar (maybe I just haven't looked hard enough), but I copy-pasted an old osu!lazer install's songs folder in hopes it would move/import them to a new installation, but it won't see any of the previous beatmaps. Is this even possible? If not, is there a way to recover the old ones to be imported in some way?

**Comment by integerrr** [comment, marked as answer]
Are you sure this is lazer you're talking about, lazer doesn't have a "songs" folder. 

Regardless, if you are trying to move the installation elsewhere there is an in game options to do so

**Reply by X-Vaskys** [reply]
Yeah, it is Lazer, obviously the folder isn't called "Songs", it's something more like "Files", but I *believe* that's where the Beatmaps are stored since it's several GB, if not then I'm unsure.

Edit: It's a fresh Windows 10 install so the only Lazer installation I have is the new one.

**Reply by bdach** [reply]
it is not sufficient to only move the files directory, the client database is also needed.

you're not supposed to be doing this manually anyway. since a few versions back you can point a new lazer installation at an older lazer installation folder by using the "choose osu! location" functionality.

**Reply by X-Vaskys** [reply]
I see; is there any way to move/obtain said Database too, or is the only option to "choose a new location" and point it at the old one?

**Reply by integerrr** [reply]
If your old install is still intact after your copy/paste then yes, lazer should now offer you to use the existing data if you point it there
(#17527)
